demo-snippet.html

```diff
@@ -55,29 +48,32 @@
 
 <dom-module id="demo-snippet">
   <template>
-    <style is="custom-style" include="prism-theme-default"></style>
```

Comment:
I had removed all the highlighting because the elements weren't ported. Adding back now